src/conspiracies/corpusprocessing/umap_hdb.py

```diff
@@ -93,7 +93,7 @@ def load_triplets(
     ]
     filtered_triplets = filter_triplets_with_stopwords(
         triplets_list,
-        get_stop_words("danish"),
+        set(get_stop_words(language)),
```

Comment:
why not just have get_stop_words return a set?

Reply:
It's from a third-party library which, for some reason, returns it as a list.

Reply:
Oh, spacy implements a stopword list as well (no need to change it just so you know)

Reply:
I'll take a look at that when I dig deeper into this part of the code.